pyanaconda/ihelp.py

```diff
@@ -22,8 +22,8 @@
 
 from pyanaconda.flags import flags
 from pyanaconda.localization import find_best_locale_match
-from pyanaconda.constants import DEFAULT_LANG
-from pyanaconda.iutil import startProgram
+from pyanaconda.core.constants import DEFAULT_LANG
+from pyanaconda.core.iutil import startProgram
```

Comment:
Can we finally remove the `i`, please?

Reply:
I'm here to serve... -> or in other words -> yep, I'll remove it.